packages/gcf-utils/src/logging-octokit-plugin.ts

```diff
@@ -15,36 +15,16 @@
 // eslint-disable-next-line node/no-extraneous-import
 import {Octokit} from '@octokit/rest';
 import {logger} from './gcf-utils';
+import {
+  OctokitRequestParser as parser,
+  GitHubActionDetails,
+  GitHubActionType,
+  GitHubObjectType,
+  OctokitRequestOptions,
+} from './octokit-request-parser';
 
 export const VERSION = '1.0.0';
 
-/**
```

Comment:
I've moved the request parsing logic from `logging-octokit-plugin` to a separate class called `OctokitRequestParser` so that it can be reused in other places